<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>570</y>
      <width>171</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="play">
       <property name="text">
        <string>SImular</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="stop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="add">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="5">
      <widget class="QDoubleSpinBox" name="radio">
       <property name="minimum">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>300.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QDoubleSpinBox" name="velx">
       <property name="minimum">
        <double>-1000000000000000013946113804119924437974165856986638331112094170909680489426130543638408513078605724209795153399497011464465488473637220910340574757582946907032347746826714825234078949864321840610832155574248213693581484614981956096327942144.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000001097906362944045541740492309677311846336810682903157585404911491537163328978494688899061249669721172515611590283743140088328307009198146046031271664502933027185697489699588559043338384466165001178426897626212945177628091195786707458122783970171784415105291802893207873272974885715430223118336.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="posy">
       <property name="minimum">
        <double>-8000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>8000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">text-decoration: underline;
font: 18pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Vel X:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;8514oem&quot;;</string>
       </property>
       <property name="text">
        <string>AGREGAR PLANTE MI PEZ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QDoubleSpinBox" name="masa">
       <property name="minimum">
        <double>20.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QDoubleSpinBox" name="vely">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-100000000000000001097906362944045541740492309677311846336810682903157585404911491537163328978494688899061249669721172515611590283743140088328307009198146046031271664502933027185697489699588559043338384466165001178426897626212945177628091195786707458122783970171784415105291802893207873272974885715430223118336.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000000000000001097906362944045541740492309677311846336810682903157585404911491537163328978494688899061249669721172515611590283743140088328307009198146046031271664502933027185697489699588559043338384466165001178426897626212945177628091195786707458122783970171784415105291802893207873272974885715430223118336.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">text-decoration: underline;
font: 18pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Radio:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">text-decoration: underline;
font: 18pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Posicion Y:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">
text-decoration: underline;
font: 18pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Posicion X:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">text-decoration: underline;
font: 18pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Vel y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">text-decoration: underline;
font: 18pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Masa:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="posx">
       <property name="minimum">
        <double>-8000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>8000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
